client/stripe-utils/utils.js

```diff
@@ -8,6 +8,8 @@ import {
 	getPaymentMethodsConstants,
 } from './constants';
 
+const { CHECKOUT_STORE_KEY, PAYMENT_STORE_KEY } = window.wc.wcBlocksData;
```

Comment:
The file `client/stripe-utils/utils.js` is imported by both the `blocks` and the `shortcode` checkouts... and is causing the E2E tests to fail because the Stripe elements do not load in the shortcode checkout

<img width="555" alt="Screenshot 2024-06-09 at 5 08 56 PM" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/407542/245dc5de-1f98-476d-b51c-151441d404a7">

Maybe we can move this into both `unblockBlockCheckout` and `resetBlockCheckoutPaymentState` functions, they each use one of the keys 🤷🏼

Reply:
👍 Sounds good. I've moved the const definitions out of file scope and in behind the `is-blocks` check. [a857630](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3187/commits/a857630fb9957c074461255c7c043975ce4ac667)